prime-dotnet-webapi/HttpClients/College Licence API/DummyCollegeLicenceClient.cs

```diff
@@ -17,7 +17,9 @@ public Task<PharmanetCollegeRecord> GetCollegeRecordAsync(string licencePrefix,
                 throw new PharmanetCollegeApiException();
             }
 
-            var info = licenceNumber switch
+            var trimmedLicenceNumber = licenceNumber.TrimStart(new char[] { '0' });
```

Comment:
`string.TrimStart()` has both a `TrimStart(char trimChar)` and a ` TrimStart(params char[]? trimChars)` overload, there is no reason to create an array here. Just use `licenceNumber.TrimStart('0')`.

Reply:
Also, you don't need `trimmedLicenceNumber`, so making a new variable here is not nessisary. just do:
```c#
var info = licenceNumber.TrimStart('0') switch
{
  // etc.
}
```